hazelcast/src/main/java/com/hazelcast/map/impl/MapServiceContext.java

```diff
@@ -102,10 +105,12 @@
 
     MapEventPublisher getMapEventPublisher();
 
-    MapQueryEngine getMapQueryEngine();
+    MapQueryEngine getMapQueryEngine(String name);
```

Comment:
This is not related to introducing a OperationFactory.

What I expect to see in this pr is all calls for operation creation to be replaced by a similar operationfactory call. That is what the scope should be.